### PR TITLE
Docs: updated resilience page

### PR DESCRIPTION
**Pull request by martijnvg**
this docs change should have been included in #7994

### CODE REVIEW

**Comment by jpountz** [comment]
LGTM

File: docs/resiliency/index.asciidoc [hunk 1]
```diff
@@ -95,7 +95,7 @@ Recovery from failure is a complicated process, especially in an asynchronous di
...
 * After joining a cluster, validate that the join was successful and that the master has been set in the local cluster state. {GIT}6969[#6969]. (STATUS: DONE, v1.4.0.Beta1)
 * Write additional tests that use the test infrastructure to verify proper behavior during network disconnections and garbage collections. {GIT}7082[#7082] (STATUS: ONGOING)
-* Make write calls return the number of total/successful/missing shards in the same way that we do in search, which ensures transparency in the consistency of write operations. {GIT}7572[#7572]. (STATUS: ONGOING)
+* Make write calls return the number of total/successful/missing shards in the same way that we do in search, which ensures transparency in the consistency of write operations. {GIT}7994[#7994]. (STATUS: DONE, fixed in 2.0.0.Beta1)
```
Comment (bleskes) [inline docs/resiliency/index.asciidoc]: I think we can remove the version?

Comment (bleskes) [inline docs/resiliency/index.asciidoc]: also, no need to say fixed. It's a feature :)

**Comment by bleskes** [comment]
LGTM. Left one small comment

File: docs/resiliency/index.asciidoc (hunk 1 above)
Comment (martijnvg) [inline docs/resiliency/index.asciidoc]: So the version gets added when a 2.0 release gets out? Or we leave it out? But then it is confusing  in what versions it was included (once 2.0 get out).

Comment (bleskes) [inline docs/resiliency/index.asciidoc]: Oh, then we should say `STATUS: DONE, v2.0.0`

**Pull request merged**